src/types/ScholarshipAmount.ts

```diff
@@ -0,0 +1,51 @@
+import AmountType from './AmountType';
+
+export default class ScholarshipAmount {
+  readonly type: AmountType;
+  readonly min: number;
+  readonly max: number;
+
+  constructor(data?: { type?: AmountType; min?: number; max?: number }) {
+    this.type = data?.type ?? AmountType.Unknown;
```

Comment:
Woah I didn't know about the `??`, pretty cool!

Reply:
Yeah it's nice!